src/SonarScanner.MSBuild.PreProcessor/SonarScanner.MSBuild.PreProcessor.csproj

```diff
@@ -13,6 +13,7 @@
     <ProjectReference Include="..\SonarScanner.MSBuild.Common\SonarScanner.MSBuild.Common.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.21.9" />
```

Comment:
Let's use `3.6.1` like on sonar-dotnet due to the performance issue we have

Reply:
The latest version does not have a performance problem.  The problem we had on sonar security was caused by the way it was used in the Roslyn context. There we had to pack the transitive dependencies and inject them at runtime.

See this PR: https://github.com/SonarSource/sonar-security/commit/e812dfea5be392a0161bdc5548a375a54164b3c3

On the scanner side, we don't run in Roslyn context so we don't interfere with it.